zealot/src/zng/types/record.ts

```diff
@@ -49,7 +53,7 @@ export class Record implements ZngClass<Type[] | null> {
   at(index: number) {
     const col = this.type[index]
     if (!col) throw new Error(`No column at index: ${index}`)
-    const val = this.value[index]
+    const val = this.value === null ? null : this.value[index]
```

Comment:
What do you think about calling `this.isSet()` and `!this.isSet()` in all the places you compare `this.value === null`. I think it might read a bit better.

Reply:
That would definitely read better.